components/Cards/index.js

```diff
@@ -17,3 +17,40 @@
 // </div>
 //
 // Create a card for each of the articles and add the card to the DOM.
+axios
+  .get(`https://lambda-times-backend.herokuapp.com/articles`)
+  .then(response => {
+    console.log(response); //pass response.articles
+    Object.values(response.data.articles).forEach(item => {
+      item.forEach(item => {
+        let cards = document.querySelector(".cards-container");
+        cards.appendChild(createCard(item));
+      });
+    });
+  });
+
```

Comment:
Great job making an axios Get request to bring in data. I like how you console logging along the way. Awesome job